fmt-12.1.0 causes dnf5 build to fail

dnf5 build fails with fmt-12.1.0. The used tag for rebuild was the latest in fedora/rawhide - dnf5-5.4.2.1-4.fc45, the encountered issue was:

(full build log is at https://kojipkgs.fedoraproject.org//work/tasks/3534/147013534/build.log )

```
FAILED: [code=1] libdnf5/CMakeFiles/libdnf5_obj.dir/base/transaction.cpp.o 
/usr/bin/g++ -DGETTEXT_DOMAIN=\"libdnf5\" -DHAVE_LIBACL -DLIBDNF_BUILD_LIBRARY -DLIBSOLV_FLAG_FOCUSNEW=1 -DLIBSOLV_SOLVABLE_PREPEND_DEP -DPROJECT_BINARY_DIR=\"/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/redhat-linux-build\" -DPROJECT_SOURCE_DIR=\"/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1\" -DPROJECT_VERSION_MAJOR=4 -DPROJECT_VERSION_MICRO=1 -DPROJECT_VERSION_MINOR=2 -DPROJECT_VERSION_PRIME=5 -DWITH_MODULEMD -I/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/include -I/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/common -I/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/libdnf5/. -I/usr/include/json-c -I/usr/include/libxml2 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-6 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m32 -march=i686 -mtune=generic -msse2 -mfpmath=sse -mstackrealign -fasynchronous-unwind-tables -fstack-clash-protection -mtls-dialect=gnu -std=gnu++20 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -fmacro-prefix-map=/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/= -Wall -Wextra -Werror -Wcast-align -Wformat-nonliteral -Wmissing-format-attribute -Wredundant-decls -Wsign-compare -Wsign-conversion -Wtype-limits -Wuninitialized -Wwrite-strings -Werror=unused-result -Wodr -Wconversion -Wl,--as-needed -MD -MT libdnf5/CMakeFiles/libdnf5_obj.dir/base/transaction.cpp.o -MF libdnf5/CMakeFiles/libdnf5_obj.dir/base/transaction.cpp.o.d -fmodules-ts -fmodule-mapper=libdnf5/CMakeFiles/libdnf5_obj.dir/base/transaction.cpp.o.modmap -MD -fdeps-format=p1689r5 -x c++ -o libdnf5/CMakeFiles/libdnf5_obj.dir/base/transaction.cpp.o -c /builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/libdnf5/base/transaction.cpp
/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/libdnf5/base/transaction.cpp: In member function ‘libdnf5::base::Transaction::TransactionRunResult libdnf5::base::Transaction::Impl::_run(std::unique_ptr<libdnf5::rpm::TransactionCallbacks>&&, const std::string&, std::optional<unsigned int>, const std::string&, bool)’:
/builddir/build/BUILD/dnf5-5.4.2.1-build/dnf5-5.4.2.1/libdnf5/base/transaction.cpp:969:116: error: conversion from ‘std::chrono::duration<long long int>::rep’ {aka ‘long long int’} to ‘time_t’ {aka ‘long int’} may change value [-Werror=conversion]
  969 |         std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch()).count());
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
cc1plus: all warnings being treated as errors
```

--- a/libdnf5/base/transaction.cpp
+++ b/libdnf5/base/transaction.cpp
@@ -967,8 +967,7 @@ Transaction::TransactionRunResult Transaction::Impl::_run(
     info.set_description(description);
     info.set_comment(comment);
     info.set_pid(getpid());
-    info.set_start_time(
-        std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+    info.set_start_time(std::chrono::system_clock::to_time_t(std::chrono::system_clock::now()));
 
     // acquire the lock
     std::filesystem::path lock_file_path = config.get_installroot_option().get_value();

